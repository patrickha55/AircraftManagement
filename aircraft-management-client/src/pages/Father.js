import React from 'react'
import Child from './Child';

const Father = () => {
    return (
        <div>
            <h1>Father</h1>
            <p>Father component</p>
            <Child />
        </div>
    )
}

export default Father