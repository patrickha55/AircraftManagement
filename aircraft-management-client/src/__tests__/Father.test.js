import Enzyme, { shallow } from 'enzyme';
import Adapter from "enzyme-adapter-react-16";
import toJson from 'enzyme-to-json';
import React from 'react';
import Father from '../pages/Father';

Enzyme.configure({ adapter: new Adapter() });

test("Father should renders", () => {
    const father = shallow(<Father />);

    expect(toJson(father)).toMatchSnapshot();
});