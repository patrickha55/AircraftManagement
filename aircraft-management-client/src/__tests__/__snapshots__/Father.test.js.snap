// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Father should renders 1`] = `
<div>
  <h1>
    Father
  </h1>
  <p>
    Father component
  </p>
  <Child />
</div>
`;
